index.bs

```diff
@@ -2330,6 +2345,25 @@ Note: Invoking this method from a [=browsing context=] where the [=Web Authentic
 
 </div>
 
+### Determining Authenticator Attachment from Authenticator Transports ### {#sctn-attachments-from-transports}
+
+When creating a {{PublicKeyCredential}} object, as the result of a [=registration ceremony|registration=] or [=authentication ceremony=], the [=client platform=] determines the [=authenticator attachment modality=] and reports it in {{PublicKeyCredential/authenticatorAttachment}}. This section defines how the [=client platform=] makes that determination.
+
+If the client platform
+<dl class="switch">
+    : cannot determine the {{transports}} used or the transport is not a member of {{AuthenticatorTransport}}, then
+    :: the [=authenticator attachment modality=] is unknown, and the value of {{PublicKeyCredential/authenticatorAttachment}} MUST be set to null.
+
+    : used an {{AuthenticatorTransport/internal}} transport, then
+    :: {{PublicKeyCredential/authenticatorAttachment}} MUST be set to "platform".
+
+    : used a {{transports|transport}} that is a member of {{transports}} other than {{AuthenticatorTransport/internal}},
+    :: {{PublicKeyCredential/authenticatorAttachment}} MUST be set to "cross-platform".
+
+Note: An [=authenticator's=] [=authenticator attachment modality|attachment modality=] could change over time.
+For example, a mobile phone might at one time only support [=platform attachment=]
+but later receive updates to support [=cross-platform attachment=] as well.
+
 ## Authenticator Responses (interface <dfn interface>AuthenticatorResponse</dfn>) ## {#iface-authenticatorresponse}
 
 [=Authenticators=] respond to [=[RP]=] requests by returning an object derived from the
```

Comment:
Below here, around approximately [lines 2735-2738](https://github.com/w3c/webauthn/pull/1668/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985L2701-L2704), we should add that the `AuthenticatorAttachment` enum is gaining a second role.

```diff
diff --git a/index.bs b/index.bs
index 139a97a..8c59fd3 100644
--- a/index.bs
+++ b/index.bs
@@ -2735,7 +2720,9 @@ attributes.
 This enumeration's values describe [=authenticators=]' [=authenticator attachment modality|attachment modalities=].
 [=[RPS]=] use this to express a preferred [=authenticator attachment modality=]
 when calling {{CredentialsContainer/create()|navigator.credentials.create()}}
-to [[#sctn-createCredential|create a credential]].
+to [[#sctn-createCredential|create a credential]],
+and [=clients=] use this to report the [=authenticator attachment modality=]
+used to complete a [=registration ceremony|registration=] or [=authentication ceremony=].

 <xmp class="idl">
     enum AuthenticatorAttachment {
```